neo/Consensus/PrepareRequest.cs

```diff
@@ -20,6 +20,10 @@ public PrepareRequest()
         {
         }
 
+        protected PrepareRequest(ConsensusMessageType msgType) : base(msgType)
```

Comment:
Since the `PrepareRequest` is no longer inherited by other classes, we can remove this constructor.

Reply:
Nice catch; I forgot about it actually.